funnel/templates/profile.html.jinja2

```diff
@@ -210,6 +209,7 @@
       </div>
     {% endif %}
 
+    {{ upcoming_section(upcoming_projects) }}
```

Comment:
One twist please: move it up only if the current user is not already a member. Existing members should see upcoming projects first.